app/models/requests/service_eligibility/on_order.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module Requests
+  module ServiceEligibility
+    # on_order - material has a status in Alma that indicates it is ordered but has not yet arrived on campus
+    class OnOrder
+      def initialize(requestable:, user:)
+        @requestable = requestable
+        @user = user
+      end
+
+      def to_s
+        'on_order'
+      end
+
+      def eligible?
+        requestable_eligible? && user_eligible?
+      end
+
+      private
+
+        def requestable_eligible?
+          requestable.on_order?
```

Comment:
Do we need any of the other conditionals (e.g. `(requestable.alma_managed? || requestable.partner_holding?) && !requestable.aeon?`, `!requestable.charged?`, `!requestable.in_process?`), or are they all redundant?

Reply:
I think they might be redundant, but just to be on the safe side I included them, since that will re-create the existing branching logic. I don't think it should be possible for on_order? to be true and those other conditions not be met, but I'm not positive, so might as well include it.